perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1844,6 +1850,27 @@ sub get_user : Private {
     return $user;
 }
 
+sub user_alert_details : Private {
+    my ( $self, $c ) = @_;
+
+    my $alerts = $c->stash->{user}->alerts;
```

Comment:
Probably want to `prefetch` with alert_type to save one query per row in the next bit. Plus stick this on the stash and use in the template because otherwise `user.alerts` will do another query both times in the template.